include/envoy/common/platform.h

```diff
@@ -300,3 +302,87 @@ struct mmsghdr {
 // On non-Linux platforms use 128 which is libevent listener default
 #define ENVOY_TCP_BACKLOG_SIZE 128
 #endif
+
+#ifndef TCP_INFO
+// Posix structure for tcp info. Currently not available to all the kernels.
+// See https://github.com/torvalds/linux/blob/master/include/uapi/linux/tcp.h
+// Some of them may remain uninitialized. If you are using this struct please refer to
+// the implementation of Windows to see that the metric you are querying for is supported.
+// The corresponding Windows is implementation is documented at:
```

Comment:
nit: `// The corresponding Windows API is documented at:`

Reply:
thanks resolved